net/ola/Makefile

```diff
@@ -62,10 +63,8 @@ HOST_CONFIGURE_ARGS += \
 	--disable-examples \
 	--disable-unittests \
 	--disable-doxygen-html \
-	--disable-doxygen-doc
-
-HOST_LDFLAGS += -Wl,-rpath$(comma)$(STAGING_DIR_HOSTPKG)/lib
```

Comment:
these have been there for a reason, pls explain in the commit message why they are removed.

Reply:
where is the description?

Reply:
> Patch ola.m4 to support statically linked protobuf. **Avoids rpath hacks**.